lib/manager/components/LabelPanel.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React, { Component } from 'react'
+import { Panel } from 'react-bootstrap'
+
+import FeedLabel from '../../common/components/FeedLabel'
+import type { ManagerUserState } from '../../types/reducers'
+
+import LabelEditorModal from './LabelEditorModal'
+
+export type Props = {
+  large?: boolean,
```

Comment:
What does `large` mean? Wide or tall?

Reply:
It means both! Although less now after feedback from Landon